resources/scripts/components/dashboard/ServerRow.tsx

```diff
@@ -53,6 +53,21 @@ export default ({ server, className }: { server: Server; className: string | und
         alarms.disk = server.limits.disk === 0 ? false : isAlarmState(stats.diskUsageInBytes, server.limits.disk);
     }
 
+    let disklimit;
+    let memorylimit;
+
+    if(server.limits.disk != 0) {
+        disklimit = bytesToHuman(server.limits.disk * 1000 * 1000);
+    } else {
+        disklimit = "Unlimited";
+    };
```

Comment:
You have the same code four times, why not wrap it into a function?
Or alternatively, maybe do the following as it's a lot shorter.
```js
const disklimit = server.limits.disk != 0 ? bytesToHuman(server.limits.disk * 1000 * 1000) : "unlimited";
```

Reply:
Used the shorter one.